### ISSUE TITLE
Formatting unspaced operators 

### ISSUE THREAD

**Issue by jfmengels** [avh4/elm-format#810]
I (too often) have code like this 

Input:
```elm
a = f>>.field
b = g|>.field
```
Actual output (version 0.8.5):
```elm
a =
    f >>. field


b =
    g |>. field
```
Expected output:

```elm
a =
    f >> .field


b =
    g |> .field
```

I think the current behavior was correct for older versions of Elm, where `>>.` was potentially a valid Elm operator. Since 0.19, the operators are all known, and my thinking is that we could decide to try to end up in a state that it is more likely correct code (but is less agnostic about the operators).

**Comment by lue-bird** [comment]
Yeah it shouldn't format code in a way that leads to compiler error

**Comment by avh4** [comment]
😆  FYI, you already opened this issue https://github.com/avh4/elm-format/issues/677

I'm adding a new label for issues that I think are good to do, but that will require complete knowledge of the possible operators (and thus will also require Elm 0.19): https://github.com/avh4/elm-format/labels/full%20operators%20list

**Comment by jfmengels** [comment]
Ugh, I looked for it too and somehow missed it :sweat_smile: 
Feel free to close whichever!
